includes/User.php

```diff
@@ -2715,23 +2702,6 @@ private function logAttributeMismatch( $valueFromMW, $valueFromService, $attribu
 	}
 
 	/**
-	 * Set an attribute local to a wikia. See createLocalOptionName for details regarding
-	 * the format. Note that the $sep param is provided in the rare case where
-	 * the option name is not normal. Should you have to use the separator, PLEASE MAKE
-	 * A PLAN TO NORMALIZE IT TO "-".
-	 *
-	 * @param string $attribute
-	 * @param string $value
-	 * @param int $cityId
-	 * @param string $sep
-	 * @return bool
-	 * @see getGlobalAttribute for more documentation about attributes
-	 */
-	public function setLocalAttribute($attribute, $value, $cityId = null, $sep = '-') {
```

Comment:
This wasn't being used. All attributes we currently store are global, if we need to start supporting wiki-specific attributes down the line we can re-add this.